x-pack/plugins/lens/common/expressions/datatable/utils.ts

```diff
@@ -5,14 +5,25 @@
  * 2.0.
  */
 
-import type { Datatable } from '@kbn/expressions-plugin/common';
+import { type Datatable, type DatatableColumnMeta } from '@kbn/expressions-plugin/common';
 import { getOriginalId } from './transpose_helpers';
 
+/**
+ * Returns true for numerical fields, excluding ranges
+ */
+export function isNumericField(meta?: DatatableColumnMeta): boolean {
+  return meta?.type === 'number' && meta.params?.id !== 'range';
+}
```

Comment:
There are two more situation that can't be considered "numeric", when the `meta.params?.id === 'filters'` and when the `params.id === 'multi_terms'`
In the filter case, the rendered cell is a string (can be anything).
The multi_terms case instead is similar to the range

Reply:
Good idea, I also added `filtered_metric` which returns an array of values. See e3609565d66f3b2416c5ebd156cfab267561a072

Reply:
great thanks. when/how `filterd_metric` is used?

Reply:
@markov00 are you good with these changes?

Reply:
good thanks